src/cmake/axom-config.cmake.in

```diff
@@ -187,10 +187,109 @@ if(NOT AXOM_FOUND)
     set_target_properties(RAJA PROPERTIES INTERFACE_COMPILE_OPTIONS "")
   endif()
 
+
+  #---------------------------------------------------------------------------
+  # Remove non-existant INTERFACE_INCLUDE_DIRECTORIES from imported targets
+  # to work around CMake error
+  #---------------------------------------------------------------------------
+
+  # Support IN_LIST operator for if()
+  # Policy added in 3.3+
+  if(POLICY CMP0057)
+    cmake_policy(SET CMP0057 NEW)
+  endif()
+
+  ## axom_find_target_dependencies(TARGET <target> TLIST <list name>)
+  ##
+  ## Store all target's dependencies (link libraries and interface link libraries)
+  ## recursively in the variable name TLIST holds.
+  macro(axom_find_target_dependencies)
+
+    set(options)
+    set(singleValuedArgs TARGET TLIST)
```

Comment:
Should `TLIST` be in `multiValuedArgs` instead of `singleValuedArgs`?

Reply:
TLIST is the name of the list variable being filled by the macro, not the contents.

Reply:
Thanks -- that makes sense